js/anxpro.js

```diff
@@ -23,14 +22,23 @@ module.exports = class anxpro extends Exchange {
             },
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/27765983-fd8595da-5ec9-11e7-82e3-adb3ab8c2612.jpg',
-                'api': 'https://anxpro.com/api',
+                'api': {
+                    'public': 'https://anxpro.com/api/2',
+                    'private': 'https://anxpro.com/api/2',
+                    'v3public': 'https://anxpro.com/api/3',
+                },
                 'www': 'https://anxpro.com',
                 'doc': [
                     'https://anxv2.docs.apiary.io',
                     'https://anxpro.com/pages/api',
                 ],
             },
             'api': {
+                'v3public': {
+                    'get': [
+                        'currencyStatic',
```

Comment:
> @kroitor I've removed all the hardcoded markets and currencies now (using a new endpoint the v3 api)

Awesome! Much better than it was before! ) One question, though: we can't list just one endpoint if we are adding an API section – can you list the rest of them (public) here?

Reply:
The v3 api literally only has one public endpoint.

It does have a load of private API calls, but I've not implemented those.

Reply:
@npomfret got it, thx for clarifying this and for the effort! Appreciate your involvement! Merging shortly!